src/tokens/wrappers/clawback/ClawbackMetadata.sol

```diff
@@ -194,4 +197,18 @@ contract ClawbackMetadata is IMetadataProvider, IERC165 {
         }
         return false;
     }
+
+    function _formatUnlocksIn(uint256 lockedAt, uint256 duration) internal view returns (string memory) {
```

Comment:
I intentionally chose not to add this as most platforms cache the metadata. Thought it would be out of date and confusing for non-native/tech people. 
Happy to include if you disagree.

Reply:
Middle ground would be to reduce the granularity to `days` and exclude hours etc.

Reply:
I don't think it is a big issue, we can update it later if it gives us trouble.